knora/dsplib/utils/excel_to_json_properties.py

```diff
@@ -1,12 +1,15 @@
 import json
 import os
+import re
 from typing import Any
 
 import jsonschema
-from openpyxl import load_workbook
+import pandas as pd
 
+from knora.dsplib.utils.excel_to_json_resources import _prepare_dataframe
```

Comment:
since you're importing this here, it probably shouldn't be private (rule of thumb: everything that's used only in the same file should be private; everything used from a different file should not)